src/qt/pivx/send.cpp

```diff
@@ -761,13 +775,15 @@ void SendWidget::onDeleteClicked(){
                 it.remove();
             } else if (focusedEntry && entry->getNumber() > entryNumber){
                 entry->setNumber(entry->getNumber() - 1);
+                entry->setFocus();
```

Comment:
Better to unify this and use the `setFocusOnLastRecipient` that you created.
Plus, move it to the bottom of the method. The last entry will get focus anyway.